alf/utils/dist_utils.py

```diff
@@ -745,6 +797,8 @@ def get_mode(dist):
             mode = base_mode
             for transform in dist.transforms:
                 mode = transform(mode)
+    elif isinstance(dist, td.Beta):
+        mode = dist.mean
```

Comment:
mode can be calculated analytically, which is (alpha -1)/(alpha+beta - 2) for alpha, beta >1. The mean is alpha/(alpha+beta). Do we need the actual mode?

Reply:
> mode can be calculated analytically, which is (alpha -1)/(alpha+beta - 2) for alpha, beta >1. The mean is alpha/(alpha+beta). Do we need the actual mode?

Yeah I think we need mode instead of mean in most cases.

Reply:
Maybe mode is better? In cases where the distribution is not uni-mode, the mean and mode could be very different?

Reply:
Changed to mode.